default/scripting/buildings/buildings.macros

```diff
@@ -31,8 +37,7 @@ SPECIES_LIKES_OR_DISLIKES_BUILDING_STABILITY_EFFECTS
                 (SpeciesContentOpinion species = LocalCandidate.Species name = ThisBuilding < 0)
             ]
             accountinglabel = "DISLIKES_BUILDING_LABEL"
-            effects =
-                SetTargetHappiness value = Value + [[STABILITY_PER_LIKED_OR_DISLIKED_BUILDING_ON_PLANET]] * SpeciesContentOpinion species = Target.Species name = ThisBuilding
+            effects = SetTargetHappiness value = Value - [[STABILITY_PER_DISLIKED_BUILDING_ON_PLANET]]
```

Comment:
It ain't an actual change because result is the same:
Because each effectsgroup was only for liked or disliked content, never both at the same time, the sign in SpeciesContentOpinion is always known, so you can just adjust the operator ('-' instead of '+' for the dislikes effects).
Should I mark it as grooming?